diff --git a/src/models/lineChart.js b/src/models/lineChart.js
index 38b9ca2..6099423 100644
--- a/src/models/lineChart.js
+++ b/src/models/lineChart.js
@@ -412,9 +412,6 @@ nv.models.lineChart = function() {
 
                 interactiveLayer.tooltip
                     .chartContainer(chart.container.parentNode)
-                    .valueFormatter(function(d,i) {
-                        return d === null ? "N/A" : yAxis.tickFormat()(d);
-                    })
                     .data({
                         value: chart.x()( singlePoint,pointIndex ),
                         index: pointIndex,
